bootable-jar/WFLY-13305_Bootable_Jar.adoc

```diff
@@ -0,0 +1,325 @@
+it setups JBoss module classloader and call into a new JBoss module “org.wildfly.bootable-jar” 
+that is in charge of the actual server run.  This new JBoss module implementation located in wildfly-core, 
+(“org.wildfly.core:wildfly-jar-runtime” artifact), is expected to be automatically provisioned by galleon in all cases. 
+If the module is not part of the provisioned server, the packaging aborts, 
+the server being packaged not supporting bootable jar packaging (eg: old release not containing new artifacts).
+
+===== Galleon provisioning
+
+Galleon provisions a server thanks to the WildFly Galleon feature-pack.
+Galleon provisioning is operated inside the maven plugin. There are 2 configuration ways:
+
+* Galleon configuration as maven plugin configuration items:
+** A Galleon feature-pack location that identifies the server and version. This is required if no provisioning.xml file is provided nor feature-packs list.
+** A list of feature-packs (if no feature-pack location has been set).
+** Optionally, a list of layers to include.
+** Optionally, a list of layers to exclude.
+* Galleon provisioning.xml file located in <app src>/galleon/provisioning.xml. NB: maven plugin configuration items (if set) override provisioning.xml file. 
+Using provisioning.xml file, custom feature-packs can be combined during the provisioning phase to provision modules, features, content (eg: DB drivers).
+
+If no galleon layers are specified, then a configuration identical to the default _standalone-microprofile.xml_ is provisioned.
+
+===== CLI script execution
+
+When building the bootable jar, a path to a set of CLI scripts can be provided to update the server configuration packaged in the bootable jar. The CLI scripts are concatenated and executed once the server is provisioned and deployment copied into it (if any).
+Having multiple CLI scripts provides enough flexibility to reuse scripts in different contexts.
+The plugin can be configured with a CLI properties file to resolve properties present in CLI scripts. 
+This is similar to the _--properties=<properties file>_ option that you can provide to JBoss CLI command line. 
+This http://www.mastertheboss.com/jboss-server/jboss-script/using-properties-in-cli-scripts[article] covers the feature.
+
+The https://github.com/wildfly-extras/wildfly-jar-maven-plugin/releases/download/2.0.0.Alpha4/index.html[community documentation] contains details on how to setup CLI scripts and CLI properties file.
+
+===== User application
+
+The war or jar main artifact the maven project is building is deployed and zipped along with the server. 
+The plugin option root-url-path=true|false (true by default) allows to rename the war file to ROOT.war 
+(doesn’t apply to other packaging). 
+
+NB: Application deployment doesn't rely on the deployment scanner. The scanner is not required it is even suggested to exclude it, useless in a bootable jar context.
```

Comment:
I'd suggest to s/`The scanner is not required it is even`/`The scanner is not required, it is even`/

Reply:
Done